ibm_catalog.json

```diff
@@ -92,11 +92,11 @@
             "diagrams": [
               {
                 "diagram": {
-                  "caption": "IBM Cloud Object Storage solution",
+                  "caption": "Object Storage - Single instance (fully configurable)",
                   "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cos/main/reference-architectures/instance.svg",
                   "type": "image/svg+xml"
                 },
-                "description": "This architecture supports creating and configuring an IBM Cloud Object Storage instance."
+                "description": "This architecture automates the provisioning and configuration of an <b>IBM Cloud Object Storage (COS) instance</b> within an existing resource group, including advanced bucket management, integration with <b>IBM Secrets Manager</b> for secure credential storage, IAM authorization policies, and enforcement of <b>Context-Based Restrictions (CBR)</b> for enhanced security and compliance."
```

Comment:
There is no bucket management in instances DA. So its not right to mention that.